app/models/testimonial.rb

```diff
@@ -0,0 +1,2 @@
+class Testimonial < ApplicationRecord
+end
```

Comment:
Don't forget to add any validation. Please note that if you have a constraint into the database table, you need to validate the same into the model.